fzf-tab.zsh

```diff
@@ -376,7 +376,9 @@ typeset -ga _ftb_group_colors=(
 () {
   emulate -L zsh -o extended_glob
 
-  fpath+=($FZF_TAB_HOME/lib)
+  if [[ ! "$FPATH" == *${FZF_TAB_HOME}/lib* ]]; then
```

Comment:
```suggestion
  if (( ! $fpath[(I)$FZF_TAB_HOME/lib] )); then
```

Reply:
That's a much more specific check, nice solution 👌